CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Deprecated
 
-- Nothing.
+- [#2](https://github.com/laminas/laminas-mvc-middleware/pull/2) Deprecates direct
+  usage of double-pass and callable middlewares. Use decorators provided by laminas-stratigility.
```

Comment:
```suggestion
- [#2](https://github.com/laminas/laminas-mvc-middleware/pull/2) deprecates direct usage of double-pass and callable middleware. Please use the decorators and helper functions provided by laminas-stratigility (`CallableMiddlewareDecorator`, `DoublePassMiddlewareDecorator`, `middleware()`, and `doublePassMiddleware()`).
```

- The plural of middleware is middleware.
- CHANGELOG notes should link to documentation, and/or provide full information on what changes a user should make.